helsingborg-stad/app-mitt-helsingborg
# Ci/builds new environment system

## Explain the changes you’ve made

Updated CI builds to work with new way of handling app environments.

## Explain why these changes are made

Simplify and unify environments and CI.

## Explain your solution

* Removed any logic regarding specific environments throughout the build workflows
* Moved secret-contents into new, non-prefixed GitHub secrets (e.g. `DEVELOP_DOTENV_CONTENTS_BASE64` -> `DOTENV_CONTENTS_BASE64`)
* Updated README

## How to test

Concrete example:

Check the Actions tab on GitHub and verify runs using this branch works. Verify TestFlight builds work as expected.
